include/api.php

```diff
@@ -3629,6 +3629,71 @@ function api_direct_messages_destroy($type)
 /// @TODO move to top of file or somewhere better
 api_register_func('api/direct_messages/destroy', 'api_direct_messages_destroy', true, API_METHOD_DELETE);
 
+function api_friendships_destroy($type)
```

Comment:
please add a doxygen header

Reply:
Ohh right! It's gone through copy&paste. Thanks

Reply:
There still isn't any Doxygen header, even though you had a commit message about it.